dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java

```diff
@@ -360,11 +402,11 @@ public boolean onTouch(View v, MotionEvent event) {
         private boolean isCancelable = true;
         private Holder holder;
         private int backgroundColorResourceId = INVALID;
-        private int gravity = Gravity.BOTTOM;
+        private Gravity gravity = Gravity.BOTTOM;
         private ScreenType screenType = ScreenType.HALF;
         private OnItemClickListener onItemClickListener;
-        private int inAnimation = R.anim.slide_in;
-        private int outAnimation = R.anim.slide_out;
+        private int inAnimation = -1;
```

Comment:
Yes, I will declare a variable like that :) Next time I will try to put more fantasy with magic numbers like 989 :D